src/EditorFeatures/Core/Wrapping/SeparatedSyntaxList/AbstractSeparatedSyntaxListWrapper.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Editor.Wrapping.SeparatedSyntaxList
+{
+    /// <summary>
+    /// Base type for all wrappers that involve wrapping a comma-separated list of items.
+    /// </summary>
+    internal abstract partial class AbstractSeparatedSyntaxListWrapper<
+        TListSyntax,
+        TListItemSyntax>
+        : AbstractWrapper
+        where TListSyntax : SyntaxNode
+        where TListItemSyntax : SyntaxNode
+    {
+        protected abstract string ListName { get; }
+        protected abstract string ItemNamePlural { get; }
+        protected abstract string ItemNameSingular { get; }
+
+        protected abstract IBlankLineIndentationService GetIndentationService();
+
+        protected abstract TListSyntax GetApplicableList(SyntaxNode node);
+        protected abstract SeparatedSyntaxList<TListItemSyntax> GetListItems(TListSyntax listSyntax);
+        protected abstract bool PositionIsApplicable(
+            SyntaxNode root, int position, SyntaxNode declaration, TListSyntax listSyntax);
+
+        public override async Task<ICodeActionComputer> TryCreateComputerAsync(
+            Document document, int position, SyntaxNode declaration, CancellationToken cancellationToken)
+        {
+            var listSyntax = GetApplicableList(declaration);
+            if (listSyntax == null)
+            {
+                return default;
```

Comment:
Sounds good.  I think this was actually because it did return a tuple at one point.  But if idoesn't now, i agree that 'null' is the appropriate resultant value.